lib/fruit/machos/header.go

```diff
@@ -204,7 +204,7 @@ func (f *machoMarkers) PatchSignature(oldHeader []byte, sigSize int64) (newHeade
 	// write signature loadcmd
 	f.patchLoadCmd(newHeader, patch, sigStart, sigSize)
 	// record patch now, buffer to be filled by caller
-	patch.Add(f.codeSize, f.sigLen, padded)
+	patch.Add(f.codeSize, sigSize, padded)
```

Comment:
```suggestion
	patch.Add(f.codeSize, f.sigLen, padded)
```

Reply:
Yep, apologies on that. I was hurrying to get the change in before I had to step out for a few :slightly_smiling_face: